dev/.rat-excludes

```diff
@@ -4,8 +4,6 @@ cache
 .gitattributes
 .project
 .classpath
-.mima-excludes
-.generated-mima-excludes
```

Comment:
After this PR https://github.com/apache/spark/commit/5b754b45f301a310e9232f3a5270201ebfc16076#diff-4426a0b8653e706c81b0c25a5a71bc325114f86ed31e96097febe79ab0fadc33L108, the `.generated-mima-excludes` file has become invalid.